### ISSUE TITLE
A11y_SQLAzureDataStudioMAC_Extensions - install_VoiceOver: When the Voice-Over focus lands on version details and Microsoft text element, its being announced as 'Microsoft,group' instead of 'text'.

### ISSUE THREAD

**Issue by bbarik06** [microsoft/azuredatastudio#19878]
**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev3;#BM_SQL AzureDataStudioMAC_June2022;#SQL Azure Data Studio;#MAC;#FTP;#DesktopApp;#VoiceOver;#AINotInScope;#WCAG4.1.2;

## Environment Details:
Application Name: SQL Azure Data Studio
Application Version: Version 1.37.0
Mac Version: macOS Monterey 12.4

## Repro Steps:

1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix.
2. Launch SQL Azure data studio.
3. Connect to a server.
4. Tab to 'Extensions' button and select.
5. Tab till 'Version' details, 'Microsoft' text.
6. Turn on Voice-Over and observe.

## Actual:
When the Voice-Over focus lands on version details and Microsoft text element, its being announced as 'Microsoft, group' instead of 'text'.

## Expected:
When the Voice-Over focus lands on version details and Microsoft text element, it should be announced as 'Microsoft, you are on a text element'.

## User Impact:
Screen readers users may find it difficult to perceive the user interface component and understand its purpose.

## Attachment:
![17_Version number](https://user-images.githubusercontent.com/98393966/176434147-ea06c6d0-fea9-48f8-8787-e8fbbfb83fd9.gif)
